drakrun/drakrun/lib/postprocessing/capa_plugin/capa_processor.py

```diff
@@ -0,0 +1,452 @@
+import functools
+import itertools
+import json
+import logging
+import multiprocessing
+import pathlib
```

Comment:
Btw. I think it would be nicer to do
```
from pathlib import Path
```

and change all occurences of `pathlib.Path` to just `Path`